src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -4939,6 +4939,22 @@ public Object xpending(final byte[] key, final byte[] groupname) {
     return client.getBinaryMultiBulkReply();
   }
 
+  @Override
+  public Object xautoclaim(byte[] key, byte[] groupName, byte[] consumerName,
+      long minIdleTime, byte[] start, XAutoClaimParams params) {
+    checkIsInMultiOrPipeline();
+    client.xautoclaim(key, groupName, consumerName, minIdleTime, start, params);
+    return client.getBinaryMultiBulkReply();
```

Comment:
Not sure, but may be?
```suggestion
    return client.getObjectMultiBulkReply();
```